app/views/faq/index.html.haml

```diff
@@ -24,3 +24,5 @@
         All questions
       - else
         %a.topic-link{href: "/faq?all=true"} All questions
+        %br
+        %a.topic-link{href: "/update_roster"} Update student roster
```

Comment:
I don't think we need a hard-coded link to this from the FAQ index; linking to it from an individual FAQ is something we can do from the frontend (by creating or editing a relevant FAQ entry) after the form is deployed.

Reply:
Oh ok. Do you have a preference of where the form should be located?

Reply:
Somewhere similar, perhaps `/update_username`, would make sense.